core/src/main/java/com/github/jensborch/webhooks/publisher/WebhookPublisher.java

```diff
@@ -70,7 +69,9 @@ private void call(final Webhook webhook, final WebhookEvent event) {
                     statusRepo.save(status.done(false));
                 })
                 .exception(e -> {
-                    LOG.warn("Error publishing to {} got error processing response", webhook, e);
+                    if (LOG.isWarnEnabled()) {
+                        LOG.warn("Error publishing to " + webhook + " got error processing response", e);
```

Comment:
It can if it is an error log but not on the warn log. The signature of the method is wrong then. It is a bit strange I agree and I actually think that they should have included the posibility to do so.

Reply:
I think it is the same for both error and warn, just took a quick look at https://github.com/qos-ch/slf4j/blob/master/slf4j-api/src/main/java/org/slf4j/helpers/AbstractLogger.java and it seems to be handled consistently, but I agree that the JavaDoc is a bit lacking. Also, to be sure I need to take a look at the right version of SL4j.

Reply:
It seem to be the same for the version we are using: https://github.com/qos-ch/slf4j/blob/v_1.7.30/slf4j-api/src/main/java/org/slf4j/helpers/MessageFormatter.java

Reply:
Actually I was wrong about the error method. That too does not have the the method either. None of the signatures exists that you are refering to.
I guess the reason is that the error is being logged anyway so no reason to guard it anyway. I still think that they should have included it because it is more pleasing to write that way.

Reply:
It is not par of the signature, but SL4J checks if last element in varargs array is an exception: https://github.com/qos-ch/slf4j/blob/0b97c416e42a184ff9728877b461c616187c58f7/slf4j-api/src/main/java/org/slf4j/helpers/MessageFormatter.java#L400

Reply:
Right it does. I think we can safely assume that people are using a version of 1.6 or later. If not we should go with the change we have here. Let me know what you think.

Reply:
I think we should assume that version 1.6+ is used